frontend/src/scenes/pipeline/destinationsLogic.tsx

```diff
@@ -109,9 +139,11 @@ export const pipelineDestinationsLogic = kea<pipelineDestinationsLogicType>([
                 },
                 deleteNodeBatchExport: async ({ destination }) => {
                     await api.batchExports.delete(destination.id)
-                    return Object.fromEntries(
-                        Object.entries(values.batchExportConfigs).filter(([id]) => id !== destination.id)
-                    )
+
+                    const batchExportConfigs = { ...values.batchExportConfigs }
+                    delete batchExportConfigs[destination.id]
+
+                    return batchExportConfigs
```

Comment:
why is this way better?

Reply:
No particular reason - just easier to parse / debug